didact/camelk/first-integration.md

```diff
@@ -0,0 +1,109 @@
+
+If you simply want to get started writing some Groovy code, you can create a folder in your workspace, create a file called `simple.groovy`, and copy in the following code:
+
+```groovy
+// camel-k: language=groovy
+from('timer:groovy?period=1s')
+    .routeId('groovy')
+    .setBody()
+        .simple('Hello Camel K from ${routeId}')
+    .to('log:info?showAll=false')
+```
+
+</details>
+
+## Step 2: Exploring the Groovy integration file
+
+Now that you have an integration file, let's take a quick look at it. If you created the file yourself, go ahead and open it now. Go to the Explorer activity (Ctrl+Shift+E) and look at the workspace folders listed.
+
+If you clicked the link to create the sample project, you can find it in `first-camelk-integration/src/simple.groovy` in the Explorer view.
+
+- [ ] If you have the `first-camelk-integration` project in your workspace, you can [open the simple.groovy file in the editor.](didact://?commandId=vscode.openFolder&projectFilePath=first-camelk-integration/src/simple.groovy&completion=Opened%20the%20simple.groovy%20file "Opens the simple.groovy file"){.didact}
+
+For this file, we're simply telling Camel to put the message `Hello Camel K from ${routeId}` in the console once every second.
+
+## Step 3: Deploying the Integration
+
+![Camel K Start Integration menu](https://raw.githubusercontent.com/camel-tooling/vscode-camelk/master/images/camelk-start-integration-popup-menu.jpg){.imageRight}
+
+The `Tooling for Camel K` extension offers several tools to get your new integration started. 
+
+First, you can right-click the `simple.groovy` file and select `Start Apache Camel K Integration`. That will provide a drop-down in the Command palette area with a number of deployment options. In this case, select the `Dev Mode - Apache Camel K Integration in Dev Mode` option. 
+
+- [ ] [Start the simple.groovy integration in Dev Mode](didact://?commandId=camelk.startintegration&projectFilePath=first-camelk-integration/src/simple.groovy&text=Dev%20Mode&completion=Deployed%20the%20simple.groovy%20file%20in%20Dev%20mode. "Deploys the simple.groovy file in 'Dev mode'"){.didact}
+
+While that starts up, we can look at the `Apache Camel K` Output channel and watch as the Camel K operator starts up the necessary resources to run our integration.
+
+## Step 4: Updating the Integration
+
+While our integration is running in `Dev Mode`, we can modify it and see those changes reflected in the deployed integration. Let's try that now.
+
+Change the message sent to the `.simple()` command of the Camel route in quotes to `We just changed our first Camel K integration while it was running!`. Save the file and see what happens in the Output channel.
+
+## Step 5: Managing our Integration
+
+We can see what integrations we currently have running in our Minikube system in the `Apache Camel K Integrations` view in the Explorer activity (Ctrl+Shift+E).
+
+- [ ] Open the `Apache Camel K Integrations` view
```

Comment:
improvement idea for another iteration: have a anction to make the Apache Camel K Integrations view visible

Reply:
Good idea for another PR. There is no generic way to do it I have found, so this sort of functionality must be implemented on a per-extension basis - https://github.com/redhat-developer/vscode-didact/issues/4